<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шашки</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>media/icon.png</normaloff>media/icon.png</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="vertical_layout" stretch="1">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_authorise"/>
    <addaction name="separator"/>
    <addaction name="action_play_english"/>
    <addaction name="action_play_russian"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_rules_for_english"/>
    <addaction name="action_rules_for_russian"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_play_english">
   <property name="text">
    <string>Играть в английские шашки</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О приложении</string>
   </property>
  </action>
  <action name="action_authorise">
   <property name="text">
    <string>Авторизоваться</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_play_russian">
   <property name="text">
    <string>Играть в русские шашки</string>
   </property>
  </action>
  <action name="action_rules_for_english">
   <property name="text">
    <string>Правила игры в английские шашки</string>
   </property>
  </action>
  <action name="action_rules_for_russian">
   <property name="text">
    <string>Правила игры в русские шашки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
